tinygrad/nn/optim.py

```diff
@@ -24,7 +24,7 @@ def realize(self, extra=None):
 class SGD(Optimizer):
   def __init__(self, params: List[Tensor], lr=0.001, momentum=0, weight_decay=0.0, nesterov=False):
     super().__init__(params)
-    self.lr, self.momentum, self.wd, self.nesterov = lr, momentum, weight_decay, nesterov
+    self.lr, self.momentum, self.wd, self.nesterov = Tensor([lr], requires_grad=False), momentum, weight_decay, nesterov
```

Comment:
Does `Tensor(lr, required_grad=False)` work? Like a 0-D tensor instead of a 1-D?